VideoLocker/src/main/java/org/edx/mobile/user/UserAPI.java

```diff
@@ -0,0 +1,44 @@
+package org.edx.mobile.user;
+
+import android.support.annotation.NonNull;
+
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.squareup.okhttp.OkHttpClient;
+
+import org.edx.mobile.discussion.RetroHttpExceptionHandler;
+import org.edx.mobile.http.RetroHttpException;
+import org.edx.mobile.util.Config;
+
+import retrofit.RestAdapter;
+import retrofit.client.OkClient;
+import retrofit.converter.GsonConverter;
+
+@Singleton
+public class UserAPI {
+
+    private UserService userService;
+
+    @Inject
+    public UserAPI(@NonNull Config config, @NonNull OkHttpClient client) {
+        Gson gson = new GsonBuilder()
+                .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+                .setDateFormat("yyyy-MM-dd'T'HH:mm:ssZ").create();
```

Comment:
Sure, moved the string to DateUtil.

On Fri, Sep 25, 2015 at 11:21 AM, Akiva Leffert <notifications@github.com>
wrote:

> In VideoLocker/src/main/java/org/edx/mobile/user/UserAPI.java
> <https://github.com/edx/edx-app-android/pull/372#discussion_r40441242>:
>
> > +import org.edx.mobile.util.Config;
> > +
> > +import retrofit.RestAdapter;
> > +import retrofit.client.OkClient;
> > +import retrofit.converter.GsonConverter;
> > +
> > +@Singleton
> > +public class UserAPI {
> > +
> > +    private UserService userService;
> > +
> > +    @Inject
> > +    public UserAPI(@NonNull Config config, @NonNull OkHttpClient client) {
> > +        Gson gson = new GsonBuilder()
> > +                .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
> > +                .setDateFormat("yyyy-MM-dd'T'HH:mm:ssZ").create();
>
> We're pretty standardized on ISO8601 for dates. Even if we don't have a
> global instance, we should probably pull the format string out?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-app-android/pull/372/files#r40441242>.
>